src/WorkItemMigrator/tests/Migration.Jira-Export.Tests/JiraLinkTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using NUnit.Framework;
+
+using JiraExport;
+using AutoFixture.AutoNSubstitute;
+using AutoFixture;
+
+namespace Migration.Jira_Export.Tests
+{
+    [TestFixture]
+    public class JiraLinkTests
+    {
+        // use auto fixiture to help mock and instantiate with dummy data with nsubsitute. 
```

Comment:
do a search replace: "auto fixiture" -> "auto fixture" in the whole project :)

Reply:
Fixed